src/js/state/QueryLibrary/selectors.ts

```diff
@@ -0,0 +1,6 @@
+import {QueryLibraryState} from "./types"
+import {State} from "../types"
+
+export default {
+  getRaw: (state: State): QueryLibraryState => state.queryLibrary
+}
```

Comment:
We may want more selectors here, and will likely return the library as a parsed tree model in one of them. For now I am only returning the raw state and the tests are all built off of that, and as I am wiring this up to the components that consume this data then I may add more